docs/core/get-started.md

```diff
@@ -25,7 +25,7 @@ dotnet run --project sample1
 
 You should see the following output:
 
-```console
+```output
```

Comment:
This should probably remain console since there is no "output" type.

```suggestion
```console
```

Reply:
We've been using output in several PRs already. Docs will simply display that value if it's a value that is not supported.

Reply:
Should we capitalize `output` so it looks nicer when we use it?

Reply:
I was asked to use lowercase by @mairaw in her review, so I did push a commit based on this.

Reply:
As we were reviewing all the PRs related to this work, we settled on only using output when it was an output from something that was not a console app. So let's use Andy's suggestion here.